src/libca/sigcert.c

```diff
@@ -368,15 +368,19 @@ int sigcert_store (const struct sigcert *cert, const char *name)
         goto error;
     if (sigcert_fwrite_public (cert, fp) < 0)
         goto error;
-    if (fclose (fp) < 0)
```

Comment:
Good catch, `-fanalyer`!

There's a small typo in the commit message: "failse"

Reply:
Thanks! Fixed and pushed.